Background.md

```diff
@@ -37,24 +39,68 @@ Een WebID duidt op meerdere 'dingen'. Het is:
 1. een document dat een 'resource' beschrijft; dat kan een beschrijving van de mens zijn, zijn profiel dus, maar ook een organisatie, een service (bot)
 1. een service voor authenticatie en deels autorisatie (OpenID Connect / OAuth2)
 
-## Verifiable Credentials (VCs)
+## Verifiable Credentials
 
-Verifiable Credentials (VCs) zijn een open standaard voor digitale attributen/referenties. Op de pagina van [W3C](https://www.w3.org/TR/vc-data-model/) kan je veel informatie vinden over VCs. Hun definitie van een VC is als volgt: "A verifiable credential is a tamper-evident credential that has authorship that can be cryptographically verified." Waarbij een credential een "set of one or more claims made by an issuer" is. Vanuit een VC kan een Verifiable Presentation worden gemaakt. 
+Verifiable Credentials (VCs) zijn een open standaard voor digitale attributen/referenties. Op de pagina van [W3C](https://www.w3.org/TR/vc-data-model/) kan je veel informatie vinden over VCs. Hun definitie van een VC is als volgt: "A verifiable credential is a tamper-evident credential that has authorship that can be cryptographically verified." Waarbij een credential een "set of one or more claims made by an issuer" is. Vanuit een VC kan een Verifiable Presentation worden gemaakt.
```

Comment:
Zou bij `#data-minimalisatie` nog iets over Verifiable Presentation genoemd moeten worden? Want dit is toch een realisatie daarvan? En dan hier een verwijzing naar dat hoofdstuk?

Reply:
Apart kopje voor toegevoegd en de verwijzing hier weggehaald